### PR TITLE
Consultancy update

### PR DESCRIPTION
**Pull request by patricia-ternes**
This PR updates the entire consultation page.

- update old text
- add a new data file to list all services provided
- include 18 new files/categories of services
  - the files were automatically created by a ruby code using the service data and one template
- include one PDF version of the service catalogue

Can someone double check if is everything working.
If the pages build, if the links work, etc...

### CODE REVIEW

File: _data/service_catalogue.yml [hunk 1]
```diff
@@ -0,0 +1,204 @@
+# Data used to create the Service Catalogue under /consulting/ page
+
+- title: Grant Proposal Support
+  url: /consulting/grant-proposal-support/
+  description: "Our support can begin at the earliest stages of your project, helping you transform research ideas into feasible computational research plans. We guide you in selecting the best techniques, tools, and infrastructure, ensuring that your grant proposal is both innovative and competitive."
```
Comment (murphyqm) [inline _data/service_catalogue.yml:5]:
```suggestion
  description: "Our support can begin at the earliest stages of your project, helping you to transform research ideas into feasible computational research plans. We will help you to select the best techniques, tools, and infrastructure, ensuring that your grant proposal is both innovative and competitive."
```

File: _data/service_catalogue.yml [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  services:
+    - item: "Research Question Feasibility"
+      description: "We collaborate with researchers to transform their research questions into feasible computational research projects. This includes assessing the problem, identifying the most suitable computational techniques, and proposing a realistic approach."
+    - item: "Techniques and Tools Identification"
+      description: "Our team can help you select the best tools, algorithms, and methods to address your research objectives effectively, ensuring that your project is both innovative and feasible."
+    - item: "Infrastructure Requirements and Planning"
+      description: "We provide expert advice on the necessary computational infrastructure, including hardware, software, and cloud resources. We also facilitate connections with relevant IT and infrastructure teams to gather the necessary information for your grant application."
```
Comment (murphyqm) [inline _data/service_catalogue.yml:12]:
```suggestion
      description: "We provide expert advice on the computational infrastructure required for your project, including hardware, software, and cloud resources. We also facilitate connections with relevant IT and infrastructure teams to gather the necessary information for your grant application."
```

File: pages/consultancy.md [hunk 3]
```diff
@@ -1,64 +1,64 @@
 ---
-title: "Research Software Engineering (RSE)"
+title: "Consultancy Support for Research"
 permalink: "/consulting/"
 breadcrumb: true
 ---
 
-The Research Software Engineering (RSE) service within Research Computing offers specialist software development consultancy, code development expertise and access to tools, platforms and training to research projects at the University of Leeds.
+Our Research Computing team at the University of Leeds, delivered through our dedicated Research Software Engineers (RSEs), offers a comprehensive range of expert services designed to support researchers throughout the entire lifecycle of their projects. You can meet members of the team via the [Team page]({{ site.baseurl }}/about/team/).
```
Comment (murphyqm) [inline pages/consultancy.md:7]: Maybe rephrase to be "Our Research Computing **service**... delivered through our RSEs" - just the "team" paired with "delivered" sounds a bit clunky to me?

**Review by murphyqm — COMMENTED** [review]
A few rephrasing comments; will test that it builds locally on Monday morning.

File: pages/consultancy.md (hunk 3 above)
Comment (patricia-ternes) [inline pages/consultancy.md:7]:
```suggestion
Our Research Computing Consultation Service at the University of Leeds, delivered through our dedicated Research Software Engineers (RSEs), offers a comprehensive range of expert services designed to support researchers throughout the entire lifecycle of their projects. You can meet members of the team via the [Team page]({{ site.baseurl }}/about/team/).
```

**Comment by patricia-ternes** [comment]
Thanks @murphyqm . I commited your suggestions.

I also changed how the consulting pages are created to improve the automation. So do an extra **pull** before test it.

Now the only "fixed" fields are `code` and `url`.
Everything else can be updated in the `_data/service_catalogue` without break any of our workflows.

If we want to be very cautious, we could include the ruby file generation step in the github action, but I don't think it is necessary.

**Comment by murphyqm** [comment]
Downloading and looking at this now to check it builds correctly.

**Comment by murphyqm** [comment]
Build working fine, no errors (once I fixed my local env); only thing I see an issue with is the alignment here:

image

**Comment by murphyqm** [comment]
> Build working fine, no errors (once I fixed my local env); only thing I see an issue with is the alignment here:
> 
> image

I'm going to approve this change anyway as I think t's important to get this information up and available; this is an aesthetic tweak that we can fix with a separate pull req.

**Review by murphyqm — APPROVED** [review]
Built locally with no errors; content looks good; content is being fed into the different sections correctly. Slight alignment issue (aesthetics), will fix in another poull req (will create na issue).

**Comment by patricia-ternes** [comment]
Thanks @murphyqm. I also had this alignment problem and I have no idea how to fix it. I agree that it is better to merge and open an issue about it
